lib/rubocop/cop/style/multiline_memoization.rb

```diff
@@ -71,6 +86,15 @@ def bad_rhs?(rhs)
             rhs.kwbegin_type?
           end
         end
+
+        def newline_after?(node, pos)
+          node.source_range.source_buffer.source[pos] == "\n"
```

Comment:
@jonas054 to make it line up do is there another autocorrect I can call or do I have to do it implement lining up here?